portfolio/src/main/webapp/life-map.js

```diff
@@ -0,0 +1,93 @@
+        'very invested in taekwondoe before me.  Taekwondo is, at a professional level, a martial arts style that ' +
+        'is characterized by fast kicking, head height kicks and other kickass techniques.  However, for third grader ' +
+        'American beginners taking lessons from a dojo next to a hair salon, it mostly meant doing "moves" around ' +
+        'a square made with duck tape in my basement for a few hours every day.  The only "cool" part if you can really ' +
+        'characterize anything that I was doing around that time as cool was breaking through plastic boards.  A fun bonus aside ' +
+        'story is in fifth grade - long after I had quit after only achieving a purple belt - my friend got really into ' +
+        'taekwondo.  When it came to board breaking the instructor held the board a little bit too high and I punched him ' +
+        'in the neck really hard by accident.  That was my last taekwondo experience. </p>' +
+        '</div>';
+
+let footballContent = '<div id="footballContent">' +
+        '<h1 id="header">Step 4: Various Activities </h1>' +
+        '<p> Since I did not in fact become the Karate (or Taekwondo?) Kid, my parents, inspired by the movie ' +
+        'remember the Titans enrolled me in football.  At this time in my life I was definitely not focused on ' +
+        'the glory of scoring a touchdown or the thrill of making a really good tackle.  Instead I was mostly ' +
+        'just going with the flow - Hukuna Matata style.  Therefore I was the bane of my "football is my life and ' +
+        'if this team doesn\'t make it to the fifth grade superbowl I am going to freak out" coach\'s life. ' +
+        'I pretty much rode the bench except for the ten minutes [mandatory to appease the parents who paid a ridiculous ' +
+        'amount of money for their fifth grader to play D-1 football].  Ironically, during this time I discovered ' +
+        'that I was really good at being coached.  Even though I never played, I ended up in the best shape of my young ' +
+        'life and was ready to actually play some sports that I enjoyed. </p>' +
+        '</div>';
+
+
+function createMap() {
+    let map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 39.577216, lng: -104.987153150},
+    zoom: 8
+    });
+
+    addLifeEvent(map, {lat: 39.577216, lng: -104.987153}, birthContent);
+    addLifeEvent(map, {lat: 39.570632, lng: -104.960932}, childhoodHomeContent);
+    addLifeEvent(map, {lat: 39.626513, lng: -104.982373}, elementarySchoolContent);
+    addLifeEvent(map, {lat: 39.579657, lng: -105.098878}, taekwondoeContent);
+    addLifeEvent(map, {lat: 39.583138, lng: -105.069205}, footballContent);
+}
+
+
+function addLifeEvent(map, eventCoordinates, eventText) {
+    let marker = new google.maps.Marker({position: eventCoordinates, map: map});
+    let info = new google.maps.InfoWindow({
```

Comment:
These can be const since you're not reassigning them anywhere.

Reply:
Made const